dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/kcp/MessageCollectorExt.kt

```diff
@@ -0,0 +1,13 @@
+package com.datadog.gradle.plugin.kcp
+
+import org.jetbrains.kotlin.cli.common.messages.CompilerMessageSeverity
+import org.jetbrains.kotlin.cli.common.messages.MessageCollector
+
+internal fun MessageCollector.abortError() {
+    report(CompilerMessageSeverity.ERROR, ERROR_MISSING_DEP)
+}
+
+private const val ERROR_MISSING_DEP =
+    "The Datadog Kotlin Compiler Plugin failed to initialize references and will abort.\n" +
```

Comment:
suggestion: let's use `Kotlin Compiler Plugin` only internally and not expose this to the customer, because it is a bit generic imo

```suggestion
    "The Datadog Compose Instrumentation failed to initialize references and will abort.\n" +
```

Reply:
updated